### ISSUE TITLE
Derive components from installer mockup

### ISSUE THREAD

**Issue by a22willi** [HGustavs/LenaSYS#14698]
Split the [installer mockup](https://github.com/HGustavs/LenaSYS/issues/14534) into smaller units/components, that can be worked on in parallel. Examples:
- BreadcrumpPanel.
- checkBoxPanel.
- ProgressBar.
- And more. 

The fonts, buttons, and smaller units that only require one css class, and are used extensively should be implemented in [#14569](https://github.com/HGustavs/LenaSYS/issues/14569). This should be for "bigger" components, that also require the use of HTML/JS/PHP. All the components should have its text/shadows/colors defined through the `installer.css` file for a more unified design language with as little repeated code as possible.

Ideally, the components should be defined as php classes, so that if we need a progress bar we can create one like this:
```
$pb = new progressBar();
$pb.setProgress(10); // set it at 10%.
$pb.setStep("Creating new database user"); // display the current step.
```

At the end of this issue, files called something like `installer_components.php` & `installer_components.js` & `installer_components.css` should be completed. Note, there should only be one of each for all components.

This is a lot of work, so it is recommend creating a separate issue for each component, so that more people can be assigned to work on the different components in parallel.

All the files should be located in the directory `newinstaller`.
